src/utils/graphql/smartweave.ts

```diff
@@ -15,9 +15,8 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 import Arweave from 'arweave';
-import { GQLNodeInterface } from 'warp-contracts';
+import { AbortError, GQLNodeInterface } from 'warp-contracts';
```

Comment:
my mistake - i thought `AbortError` was available natively, let's create our own and throw it rather than import warps

Reply:
https://github.com/ar-io/ar-io-sdk/pull/105/commits/1e2ed9168e31c58a5410213818731ab8a94aa5d2